impl/src/rendering/templates/workspace.BUILD.template

```diff
@@ -4,6 +4,13 @@ package(default_visibility = ["//visibility:public"])
 licenses([
     "notice",  # See individual crates for specific licenses
 ])
+{%- if is_remote_mode %}
+
+# Export file for Stardoc support
+exports_files([
```

Comment:
Can this be deduplicated against the inlined contents in the render function? This duplication seems crappy

Reply:
Deduplicated it